pytorch_translate/onnx_component_export.py

```diff
@@ -17,7 +17,7 @@ def get_parser_with_args():
         description=("Export PyTorch-trained FBTranslate models to Caffe2 components")
     )
     parser.add_argument(
-        "--checkpoint",
+        "--path", "--checkpoint",
```

Comment:
Are you sure about this?

Reply:
This change was originally made as a part of https://fburl.com/diff/e39xz371 - but was partially reverted in https://fburl.com/diff/3tptltgo without changing the use of args.path below